### PR TITLE
Frequency Instant sealing #234

### PR DESCRIPTION
**Pull request by saraswatpuneet**
# Goal
The goal of this PR is  to create instant-sealing mode for frequency for it to run standalone in dev mode

# Checklist
- [x] Chain spec updated
- [x] Implemented instant sealing route for frequency
- [x] Update scripts and readme
- [x] Run full e2e tests to ensure instant sealing mode is working

image

### CODE REVIEW

**Comment by saraswatpuneet** [comment]
@wilwade  this works i tried RPC calls, however polkadotjs wont pick it up

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.PHONY: start
+start:
+	./scripts/init.sh start-frequency-instant
+
```
Comment (saraswatpuneet) [inline Makefile:4]: make file to simplify interaction with Frequency
make start will start dev mode with instant-sealing

File: node/cli/src/cli.rs [hunk 2]
```diff
@@ -66,6 +66,12 @@ pub struct Cli {
 	/// Relay chain arguments
 	#[clap(raw = true)]
 	pub relay_chain_args: Vec<String>,
+
+	/// Instant block sealing
+	///
+	/// Can only be used with `--dev`
+	#[clap(long = "instant-sealing", requires = "dev")]
+	pub instant_sealing: bool,
 }
```
Comment (saraswatpuneet) [inline node/cli/src/cli.rs:75]: flag for instant seal mode

File: node/service/src/service.rs [hunk 3]
```diff
@@ -536,3 +552,172 @@ pub async fn start_parachain_node(
 	)
 	.await
 }
+fn frequency_dev_instant(config: Configuration) -> Result<TaskManager, sc_service::error::Error> {
```
Comment (saraswatpuneet) [inline node/service/src/service.rs:555]: function that runs authorship as manual seal and starts frequency as standalone not needing validators

**Comment by saraswatpuneet** [comment]
image

**Review by wilwade — APPROVED** [review]
Works for me. We'll do another PR with README updates once all the specs are done.

**Pull request merged**